modules/oe_migration_workflow/tests/src/Unit/Plugin/migrate/process/SetWorkflowStateTest.php

```diff
@@ -0,0 +1,308 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_migration_workflow\Unit\Plugin\migrate\process;
+
+use Drupal\Core\Config\ConfigManagerInterface;
+use Drupal\Core\Config\Entity\ConfigEntityTypeInterface;
+use Drupal\Core\Entity\EntityStorageInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\oe_migration_workflow\Plugin\migrate\process\SetWorkflowState;
+use Drupal\migrate\Row;
+use Drupal\Tests\migrate\Unit\process\MigrateProcessTestCase;
+use Drupal\workflows\WorkflowInterface;
+use stdClass;
```

Comment:
?

Reply:
haha stupid mistake declaring a dummy class for a test. Removed.